pkg/apis/v1/nodepool_conversion_test.go

```diff
@@ -269,6 +270,24 @@ var _ = Describe("Convert V1 to V1beta1 NodePool API", func() {
 			Expect(v1beta1nodepool.Status.Resources[resource]).To(Equal(v1nodepool.Status.Resources[resource]))
 		}
 	})
+	Context("Round Trip", func() {
+		DescribeTable(
+			"NillableDuration",
+			func(value string, field func() *NillableDuration) {
```

Comment:
Why do you need field to be a function that returns a NillableDuration rather than the pointer itself? I think this indirection might be creating some confusion here with reading through it. Also, considering the indirection here -- you might be better off just making these separate It() statements rather than a DescribeTable()

Reply:
Ya, I don't think the indirection is worth it here. I'm going to separate it out.